### DISCUSSION TITLE
Short Balance placeholders

### DISCUSSION THREAD

**Discussion by x1p** [Zrips/CMI#7092] [Enhancement]
I use all money related placeholders in game are formatted normally like: `€ 1234,50`
However, in the scoreboard, I would like to use `€ 1,2k` just to keep it simple.

The only options is to use the `%vault_eco_balance_formatted%` placeholder.
On the placeholder page https://www.zrips.net/cmi/placeholders/ it's marked are `Deprecated`
So ideally I would have to use any of the `%cmi_user_balance(_formatted)` placeholders.
Any of these result in `€ 1234,5` or `1234.5`

Short numbers are only possible in CMI (natively) if set in the config `Economy.Global.UseShortNumbers` to true
This turns everything into short numbers and there is no escaping.

I would like to have normal numbers by default, but with the possibility to use short numbered when needed.
Therefore I would like to suggest to add a placeholder something like `%cmi_user_balance_shortformated%`

**Comment by mrfloris** [comment]
This would be rather nice, indeed. +1

**Comment by Ricozaur** [comment]
Yes
